lib/VisualBuilder.js

```diff
@@ -665,7 +667,7 @@ class VisualBuilder extends EventEmitter {
                 name: 'RScript',
                 handler: () => new Promise((resolve) => resolve())
             },
-            'stringResources/**/*.resjson': {
+            '.resjson': {
```

Comment:
What about ```.json``` files? As far as I understand, it breaks back compatibility with the previous version of PBIVIZ. Is that correct?